es6-operators/main.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-unused-vars */
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+
+const checkv1 = v1 !== false ? console.log('v1 is truthy') : false;
+const checkv2 = v2 === false ? console.log('v2 is truthy') : true;
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
+ * In one expression, log "v1 is truthy" to the console only if `v1` is truthy.
+ * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
+ * Expected: v1 message logs, v2 message does not.
+ */
+
+const config1 = v1 !== true ? v1 : 'default value';
+const config2 = v2 === false ? v2 : 'default value';
+console.log('config1 =', config1, 'config2 =', config2);
+/* TODO:
+ * Demonstrate how to assign a "default value" to a variable.
+ * In one expression, assign `'default-value'` to the variable `config1` if v1 is falsy.
+ * In one expression, assign `'default-value'` to the variable `config2` if v2 is falsy.
+ * Log both `config1` and `config2`.
+ * Expected: `config1 = { value: 'Something' }`, `config2 = 'default-value'`.
+ */
+
+const cfg1 = v1 || 'default value';
+const cfg2 = v2 || 'default value';
+const cfg3 = v2 || 'default value';
+console.log('config1 =', cfg1, 'cfg2 =', cfg2, 'cfg3 =', cfg3);
+/* TODO:
+ * Demonstrate how to assign a "default value" to a variable only if the variable is `undefined`.
+ * In one expression, assign `'default-value'` to the variable `cfg1` if v1 is undefined.
+ * In one expression, assign `'default-value'` to the variable `cfg2` if v2 is undefined.
+ * In one expression, assign `'default-value'` to the variable `cfg3` if v2 is undefined.
+ * Log `cfg1`, `cfg2`, and `cfg3`.
+ * Expected: `config1 = { value: 'Something' }`, `config2 = ''`, `config3 = 'default-value'`.
+ */
```

Comment:
Make sure you're using the correct operator for these. Since `v2` is not `undefined`, `cfg2` should equal `v2`:

---

![image](https://github.com/annaaksenov/m2-0723-code-solutions/assets/16143771/9ae2e3df-dd1c-4a1e-8ca2-228b9cc267cb)